### PR TITLE
added darkmode in signuplogin page

### PR DESCRIPTION
**Pull request by Srit20**
Closes: #1841
### Describe the changes you've made
added darkmode in signuplogin page

### Screenshots
Screenshot (368)
Screenshot (367)

### CODE REVIEW

File: signuplogin.html [hunk 1]
```diff
@@ -18,6 +18,20 @@
...
+			<div>
+				<input type="checkbox" class="checkbox" id="chk" />
+				<label class="label" for="chk">
+				  <i class="fas fa-moon"></i>
+				  <i class="fas fa-sun"></i>
+				  <div class="ball"></div>
+				</label>
+			  </div>
+	
+	</nav>
```
Comment (Rizwan-S) [inline signuplogin.html:33]: Something is wrong with the code of toggle button. The changes are not reflected and hence the button is not visible.

**Review by Rizwan-S — CHANGES_REQUESTED** [review]

**Comment by Vishal-raj-1** [comment]
@Srit20 Any update from your side ?

**Comment by Srit20** [comment]
sir i have already done it 4 days ago #1841

**Comment by Srit20** [comment]
sir i have already done it 4 days ago and it is closed by #1871

**Pull request closed without merging**